lib/posticle/src/grammar.pest

```diff
@@ -118,7 +118,7 @@ terminator = _{
 }
 
 punctuation = _{
-    terminator | "-" | "_" | "#" | "+" | "/" | "^" | "`" | "{" | "}" | "‹" | "›" | "»" | "«" | "「" | "」" | "《" | "》" | "‘" | "’" | "‚" | "‛" | "“" | "”" | "„" | "‟" | "…"
+    terminator | DASH_PUNCTUATION | OPEN_PUNCTUATION | CLOSE_PUNCTUATION | INITIAL_PUNCTUATION | FINAL_PUNCTUATION | "#" | "+" | "/" | "^" | "`" | "…"
```

Comment:
Oh nice! I wonder if those were added in v2, or if I just missed them somehow when using v1...

`LETTER` solves my worries about writing rules for matching regex `\pL` too. This expands the horizons a bit.

Reply:
are there changes you'd like to make as a result? if there aren't, i was planning on merging the pr!

Reply:
No, we're good, this would just make it a lot easier to do the strict URL parsing, but that's handled right now by the validators, and that to be honest makes the most sense for portability, etc.

I'll work on #117 next.